opam-health-check.opam

```diff
@@ -48,14 +45,27 @@ depends: [
   "x509" {>= "1.0.0"}
   "docker_hub" {>= "0.1.0" & < "0.2.0"}
   "memtrace" {>= "0.2.3"}
-  "tls-lwt" {>= "0.16.0"} # activate conduit with TLS for slack webhooks
-  "conf-libev" # Required for lwt to avoid hitting the limits of select(2)
-  "conf-pixz" # TODO: Make it a library
-  # TODO: Add conf-ugrep
+  "tls-lwt" {>= "0.16.0"}
+  "conf-libev"
+  "conf-pixz"
+  "odoc" {with-doc}
 ]
 conflicts: [
-  "dns-client-lwt" {< "6.1.4"} # Required to fix https://github.com/roburio/http-lwt-client/issues/8
-  "happy-eyeballs-lwt" {< "0.1.3"} # Required to fix https://github.com/roburio/http-lwt-client/issues/8
+  "dns-client-lwt" {< "6.1.4"}
+  "happy-eyeballs-lwt" {< "0.1.3"}
 ]
-available: os-distribution != "fedora" # No pixz
```

Comment:
We still need to exclude Fedora as `conf.pixz` cannot be installed on Fedora as it isn't available.

Add `opam-health-check.template` with
```
available: os-distribution != "fedora"
```